src/Common/src/System/SR.vb

```diff
@@ -92,3 +103,23 @@ Namespace System
         End Function
     End Class
 End Namespace
+
+Namespace System.Resources.Diagnostics
+
+    <AttributeUsage(AttributeTargets.All)>
+    <ExcludeFromCodeCoverage>
+    Friend Class ExcludeFromCodeCoverageAttribute
+        Inherits Attribute
+
+        ' The code in the partial SR above is injected into all assemblies with resources, regardless of
+        ' whether those assemblies use this functionality.  As such, it should be excluded from code coverage.
+        ' The code coverage tools are configured to ignore any code attributed with an attribute
+        ' named ExcludeFromCodeCoverage, regardless of its namespace.  We have it in a specialized namespace
+        ' so as to avoid conflicts with other ExcludeFromCodeCoverageAttribute types used elsewhere in corefx.
+        ' It's applied to the individual members in SR rather than to SR as a whole because we still
+        ' want code coverage to include the individual resource keys; doing so helps to highlight whether
+        ' we're exercising all error paths, whether any resource strings are stale and can be removed, etc.
+
+    End Class
+
+End Namespace
```

Comment:
do we need to exclude only these methods or the whole SR class. I am just wondering if we need to set the attribute on the whole class instead. 

but in general the change looks good to me.